tests/utils/test_clear.py

```diff
@@ -0,0 +1,176 @@
+            "description": "One subscription to rule them all",
+            "subject": {
+                "entities": [
+                    {
+                        "idPattern": ".*",
+                        "type": "Room"
+                    }
+                ],
+                "condition": {
+                    "attrs": [
+                        "temperature"
+                    ],
+                    "expression": {
+                        "q": "temperature>40"
+                    }
+                }
+            },
+            "notification": {
+                "http": {
+                    "url": "http://localhost:1234"
+                },
+                "attrs": [
+                    "temperature",
+                    "humidity"
+                ]
+            },
+            "expires": datetime.now(),
+            "throttling": 0
+        })
+        self.cb_client.post_subscription(subscription=subscription)
+        clear_context_broker(url=self.cb_url, fiware_header=self.fiware_header)
+
+        self.assertEqual(0, len(self.cb_client.get_entity_list()) or len(self.cb_client.get_entity_list()))
+
+    def test_clear_iot_agent(self):
+        """
+        Test for clearing iota using iota client
+        """
+        service_group = ServiceGroup(entity_type='Thing',
+                                     resource='/iot/json',
+                                     apikey=str(uuid4()))
+        device = {
+            "device_id": "test_device",
+            "service": self.fiware_header.service,
+            "service_path": self.fiware_header.service_path,
+            "entity_name": "test_entity",
+        }
+
+        self.iota_client.post_groups(service_group, update=False)
+        self.iota_client.post_device(device=Device(**device), update=False)
+
+        clear_iot_agent(iota_client=self.iota_client)
+
+        self.assertEqual(0, len(self.iota_client.get_device_list()) or len(self.iota_client.get_group_list()))
+
+    def test_clear_iot_agent_url(self):
+        """
+        Test for clearing iota using iota url and fiware header as parameters
+        """
+        service_group = ServiceGroup(entity_type='Thing',
+                                     resource='/iot/json',
+                                     apikey=str(uuid4()))
+        device = {
+            "device_id": "test_device",
+            "service": self.fiware_header.service,
+            "service_path": self.fiware_header.service_path,
+            "entity_name": "test_entity",
+        }
+
+        self.iota_client.post_groups(service_group, update=False)
+        self.iota_client.post_device(device=Device(**device), update=False)
+
+        clear_iot_agent(url=self.iota_url, fiware_header=self.fiware_header)
+
+        self.assertEqual(0, len(self.iota_client.get_device_list()) or len(self.iota_client.get_group_list()))
+
+    def test_clear_quantumleap(self):
+        # TODO
+        pass
```

Comment:
Could you please add quantumleap test?

Reply:
I must be missing something, because i already see a quantumleap test